akka-bbb-apps/build.sbt

```diff
@@ -36,6 +37,10 @@ retrieveManaged := true
 
 libraryDependencies += "org.scalatest" %% "scalatest" % "3.0.8" % "test"
 libraryDependencies += "org.scala-lang.modules" %% "scala-xml" % "2.0.0"
+//libraryDependencies ++= Seq(
```

Comment:
If this lib is relevant for the future, please add some comment before it explaining the purpose and why it's not yet the time for it. Remove otherwise please.

Reply:
This library is not relevant. I was just testing to see whether it was required or not and forgot to remove the commented out import statement.